benchmarks/src/main/java/com/google/samples/apps/nowinandroid/interests/InterestsActions.kt

```diff
@@ -48,3 +48,17 @@ fun MacrobenchmarkScope.interestsToggleBookmarked() {
     checkable.click()
     device.waitForIdle()
 }
+
+fun MacrobenchmarkScope.setAppTheme(isDark: Boolean) {
```

Comment:
I'm not sure why this function is inside `InterestsActions`. The Settings button can be found on every screen in the app inside the top app bar. It might make more sense to have this in a `ForYouActions` file (because ForYou is the first screen in the app), or just `AppActions`.